### ISSUE TITLE
Mapping tables are not scaffolding

### ISSUE THREAD

**Issue by swathi1111** [PomeloFoundation/Pomelo.EntityFrameworkCore.MySql#166]
### Problem
With scaffolding mapping tables which are created for many-to-many relationship with only foreign key relationships, are ending up with error. Is there any alternative to work with such classes manually?

### The issue
Context class generating as
//Unable to generate entity type for table 'table_map'. Please see the warning messages.

Pomelo.EntityFrameworkCore.Mysql 1.1.0 version has been used

**Comment by Kagamine** [comment]
Please provide me your db creation script.

**Comment by swathi1111** [comment]
Tables are as below
**Role table:** 

```
CREATE TABLE `role` (
  `ID` int(11) NOT NULL,
  `Role` varchar(45) NOT NULL,
  `Description` varchar(45) DEFAULT NULL,
  PRIMARY KEY (`ID`)
) 
```
**User table:**
```
CREATE TABLE `user` (
  `ID` int(11) NOT NULL,
  `FirstName` varchar(100) NOT NULL,
  `LastName` varchar(100) NOT NULL,
  `AddressLine1` varchar(200) NOT NULL,
  `AddressLine2` varchar(200) DEFAULT NULL,
  `City` varchar(45) NOT NULL,
  `Country` varchar(45) NOT NULL,
  PRIMARY KEY (`ID`)
)
```
**User Role Map table:**
```
CREATE TABLE `userrole` (
  `UserID` int(11) NOT NULL,
  `RoleID` int(11) NOT NULL,
  KEY `FK_UserRole_UserID_idx` (`UserID`),
  KEY `FK_UserRole_RoleID_idx` (`RoleID`),
  CONSTRAINT `FK_UserRole_RoleID` FOREIGN KEY (`RoleID`) REFERENCES `role` (`ID`) ON DELETE NO ACTION ON UPDATE NO ACTION,
  CONSTRAINT `FK_UserRole_UserID` FOREIGN KEY (`UserID`) REFERENCES `user` (`ID`) ON DELETE NO ACTION ON UPDATE NO ACTION
)
```

_UserRole_ is a map table contains only foreign key relationships to the _User_ and _Role_ tables.

During scaffolding map table ending up with no relationship to the actual tables and resulted as below in the context class.
//Unable to generate entity type for table 'userrole'. Please see the warning messages.

**Comment by sauravMSFT** [comment]
Yes, I am facing this same issue. But after going through MySQL concepts again, Weak entities do need to define the combination of those foreign keys as their own primary key, right?

**Comment by Kagamine** [comment]
I confirmed mapping entity could be scaffolded fine in sqlserver. I will try to fix this next days.

**Comment by caleblloyd** [comment]
@Kagamine can this be done by September 4th for the 2.0.0 release or should we punt this to 2.1?

**Comment by Kagamine** [comment]
@swathi1111 Did you created more than one database with asp.net identity? If yes, should be fixed in #414.
